storage/Devices/BcacheImpl.cc

```diff
@@ -232,6 +297,48 @@ namespace storage
     }
 
 
+    CacheMode
+    Bcache::Impl::get_cache_mode() const
+    {
+        return cache_mode;
+    }
```

Comment:
Such simple functions can go in the header file and be inlined.

Reply:
Ok, makes sense. I expect that explicit `inline` keyword is no longer needed with modern compilers?

Reply:
AFAIK it was never required for functions implemented directly in the class definition and for functions implemented outside always was.